spec/unit/actions/synchronize_cas_spec.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+describe GoodData::LCM2::SynchronizeComputedAttributes do
+  let(:gdc_gd_client) { double(GoodData::Rest::Client) }
+  let(:logger) { double(Logger) }
+  let(:project) { double(GoodData::Project) }
+  let(:synchronize) do
+    [{ from: 'foo', to: [{ ca_scripts: { 'maqlDdlChunks' => ['expected_maql'] } }] }]
+  end
+  let(:basic_params) do
+    {
+        gdc_gd_client: gdc_gd_client,
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.